package.json

```diff
@@ -19,9 +20,10 @@
     "url": "https://github.com/damianavila/RISE.git"
   },
   "scripts": {
+    "build": "for target in build-css build-reveal reset-reveal; do npm run $target; done",
     "build-css": "npm run less",
-    "build-reveal": "cp -r ./node_modules/reveal.js rise/static/reveal.js",
-    "reset-reveal": "sed -i.upstream '11 s_^_\/*_' rise/static/reveal.js/css/reveal.css",
+    "build-reveal": "for dep in reveal.js reveal.js-chalkboard; do cp -r ./node_modules/$dep rise/static/$dep; done",
+    "reset-reveal": "sed -i.upstream '11 s_^_/*_' rise/static/reveal.js/css/reveal.css",
```

Comment:
There is a missing backslash on this line, are you testing this with OSX, right?
I would prefer to keep the backslash because it is harmless for OSX and I have tested successfully on linux, in the past...

Reply:
Eventually, we can use the `reveal_rise` package I created for the upcoming jupyterlab_rise, ref https://github.com/damianavila/RISE/pull/381#issue-186469852 to provide the proper CSS and get rid of this hack...